src/lib/Microsoft.Health.Logger/Telemetry/TelemetryExtensions.cs

```diff
@@ -8,12 +8,16 @@
 using Microsoft.ApplicationInsights;
 using Microsoft.ApplicationInsights.DataContracts;
 using Microsoft.ApplicationInsights.Metrics;
+using Microsoft.Health.Common.Extension;
 
 namespace Microsoft.Health.Logging.Telemetry
 {
     public static class TelemetryExtensions
     {
         private static readonly string _namespace = MetricIdentifier.DefaultMetricNamespace;
+        private static readonly string _helplinkAttribute = "helpLink";
+        private static readonly string _messageAttribute = "message";
+        private static readonly string _logToCustomerAttribute = "logToCustomerIot";
```

Comment:
Just curious why we need to add the IoT suffix? The column could be used by other services too?

Reply:
Yes, we are sharing the input source. 
An alternative is to use both this attribute and the resource type to filter, but that would complicate the mds agent's query and I don't know where exactly that query is processed or if it can overwhelm the agent.

Reply:
Are the properties visible in the logs on the customer's subscription or only used internally for filtering?

Reply:
They are used internally, the visible fields are specified in the query in the paas PR